static/app/utils/discover/fields.tsx

```diff
@@ -1,5 +1,6 @@
 import {LightWeightOrganization, SelectValue} from 'app/types';
 import {assert} from 'app/types/utils';
+import {ADOPTION_STAGES} from 'app/views/releases/list/releaseHealth/content';
```

Comment:
I would probably move this to somewhere in `app/constants` since it is used outside of `app/views/releases`. if it's an export used outside of its view, it should be pulled up